api/csharp/pubsub-publish/src/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Threading.Tasks;
+using Google.Protobuf;
+using Grpc.Core;
+using Grpc.Net.Client;
+
+namespace toit.demos.api.csharp.pubsubpublish
+{
+    public class Program
+    {
+        private static string apikey = "<apikey>"; // From: https://console.toit.io/project/apikeys
+        static async Task Main(string[] args)
+        {
+            var callCredentials = CallCredentials.FromInterceptor(async (context, metadata) =>
+            {
+                await Task.Delay(100);
```

Comment:
I don't think you need this.

Reply:
Just to not get a compiler warning, but I've changed it to return a Task.CompletedTask instead